osclients.py

```diff
@@ -31,12 +31,24 @@
 from keystoneclient.v2_0 import client as keystonev2
 from keystoneclient.v3 import client as keystonev3
 
-from neutronclient.v2_0 import client as neutronclient
-from novaclient import client as novaclient
-from cinderclient.v2 import client as cinderclient
-from glanceclient import client as glanceclient
-from swiftclient import client as swiftclient
-
+if 'OSCLIENTS_MODULES' not in env:
```

Comment:
Instead of using a new environment variable to manage imports, perhaps we could import the necessary lib when user is calling the specific method to get the client. I mean, for instance, when user calls to `osclient.get_swiftclient`, in that moment we import the _python-swiftclient_.